frontend/src/pages/notebookController/screens/server/StartServerModal.tsx

```diff
@@ -167,7 +167,7 @@ const StartServerModal: React.FC<StartServerModalProps> = ({
 
   const renderButtons = () =>
     !isNotebookRunning ? (
-      <Button key="cancel" variant="secondary" onClick={onClose}>
+      <Button key="cancel" variant={spawnFailed ? 'primary' : 'secondary'} onClick={onClose}>
```

Comment:
Small UX request by Kyle... snuck it in here, felt weird making a PR for a single line change.

Reply:
Caught an example of this happening...

Solved itself, but it went through this cycle:
![Screen Shot 2022-08-18 at 11 15 29 AM](https://user-images.githubusercontent.com/8126518/185432199-274581da-85e6-4311-905f-758cf7a5f723.png)
![Screen Shot 2022-08-18 at 11 15 38 AM](https://user-images.githubusercontent.com/8126518/185432200-a7342116-eb75-44bd-801d-07755de195c8.png)